lxc/lxc-openvdc/lxc-openvdc.go

```diff
@@ -0,0 +1,216 @@
+package main
+
```

Comment:
Can we move the folder to ``cmd/lxc-openvdc``? And I recommend to rename this file to ``main.go``